Tests/configure_and_test_integration_instances.py

```diff
@@ -41,7 +41,7 @@
     'oproxy.demisto.ninja',
     'oproxy-dev.demisto.ninja',
 ])
-NO_PROXY_CONFIG = {'python.pass.extra.keys': f'--env##no_proxy={NO_PROXY}'}  # noqa: E501
+NO_PROXY_CONFIG = {'python.pass.extra.keys': f'--env##no_proxy={NO_PROXY}##--network=host'}  # noqa: E501
```

Comment:
if i understand correctly, this also affects instances that run with docker
shouldnt we apply it only to instances with podman?

Reply:
It should be only with podman